[2025-07-13T16:08:04.532+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-13T16:08:04.569+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: employee_ingestion_dag.send_alerts scheduled__2025-07-13T16:07:00+00:00 [queued]>
[2025-07-13T16:08:04.581+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: employee_ingestion_dag.send_alerts scheduled__2025-07-13T16:07:00+00:00 [queued]>
[2025-07-13T16:08:04.581+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-07-13T16:08:04.595+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): send_alerts> on 2025-07-13 16:07:00+00:00
[2025-07-13T16:08:04.603+0000] {warnings.py:112} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=734) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-13T16:08:04.606+0000] {standard_task_runner.py:72} INFO - Started process 738 to run task
[2025-07-13T16:08:04.606+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'employee_ingestion_dag', 'send_alerts', 'scheduled__2025-07-13T16:07:00+00:00', '--job-id', '366', '--raw', '--subdir', 'DAGS_FOLDER/ingestion_pipeline.py', '--cfg-path', '/tmp/tmp48c8umr3']
[2025-07-13T16:08:04.607+0000] {standard_task_runner.py:105} INFO - Job 366: Subtask send_alerts
[2025-07-13T16:08:04.670+0000] {task_command.py:467} INFO - Running <TaskInstance: employee_ingestion_dag.send_alerts scheduled__2025-07-13T16:07:00+00:00 [running]> on host a0a54bfbbb9e
[2025-07-13T16:08:04.760+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='employee_ingestion_dag' AIRFLOW_CTX_TASK_ID='send_alerts' AIRFLOW_CTX_EXECUTION_DATE='2025-07-13T16:07:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-13T16:07:00+00:00'
[2025-07-13T16:08:04.761+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-13T16:08:04.777+0000] {ingestion_pipeline.py:219} INFO - HTML report generated: /opt/airflow/great_expectations/uncommitted/data_docs/local_site/validations/validation_report_20250713_160804.html
[2025-07-13T16:08:05.861+0000] {ingestion_pipeline.py:254} INFO - Teams alert sent successfully
[2025-07-13T16:08:05.862+0000] {ingestion_pipeline.py:261} INFO - 
            ðŸš¨ DATA VALIDATION ALERT ðŸš¨
            File: employee_data2_with_errors_chunk_12.csv
            Total Rows: 2417
            Valid Rows: 2417
            Invalid Rows: 0
            Criticality: LOW
            Issues: No issues found
            Report: http://localhost:8080/static/validation_reports/validation_report_20250713_160804.html
            
[2025-07-13T16:08:05.864+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-13T16:08:05.879+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-13T16:08:05.881+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=employee_ingestion_dag, task_id=send_alerts, run_id=scheduled__2025-07-13T16:07:00+00:00, execution_date=20250713T160700, start_date=20250713T160804, end_date=20250713T160805
[2025-07-13T16:08:05.955+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-13T16:08:05.975+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-13T16:08:05.977+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
